cpp-tests/tests/uniffi_docstring/main.cpp

```diff
@@ -0,0 +1,48 @@
+#include "test_common.hpp"
+
+#include <fstream>
+
+#include <uniffi_docstring.hpp>
+
+const std::vector<std::string> expected_docstrings = {
+    "<docstring-namespace>",
```

Comment:
Is it somehow possible to extract this list programatically from [test_generated_bindings.rs](https://github.com/mozilla/uniffi-rs/blob/42362c8fdd7eadb4f94da7945f22923f7dc5059a/fixtures/docstring/tests/test_generated_bindings.rs#L14)? I understand its more complicated than with C# where code is available as submodule, but it would be nice to give it some thought. This list is quite large, and it would be easy to forget to update it in the future.

Reply:
Without transitioning to a submodule i think the only thing that could be done is employ a stricter checking, where it would fail if there were more docstring instances than in the list, then we'd be forced to update it